notebooks/benchmark/simple_benchmark.ipynb

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Well, it's mostly just because the performance monitoring feature I added first was specifically for the notebooks. I also thought tests would be too granular if we had a separate output file for each test (or the opposite problem if it was one output per pytest command). I didn't even think about just using regular python files, though. That would seem to make sense, do you think it's preferred to notebooks?

Reply:
It would certainly be simpler to setup. :-)

I don't think there's anything notebook-specific that we need to benchmark, is there?

Reply:
Makes sense, we'll switch to regular python files for the benchmark tests then. Thanks!